### ISSUE TITLE
[A11Y] Android: Some of the components section are categorized as Buttons when focused is moved on overall frame of a respective component

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#403]
**ENVIRONMENT:**

OS (number): N (7.0)
Build number: NRD91R
Form factor (Smartphone / tablet): Nexus 9
Browser: Chrome Dev 59.0.3062.

**PRE-REQUISITE**

1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly Navigate to COMPONENTS link and double tap on it.
3. Try to Linearly navigate through different COMPONENTS link

**ACTUAL RESULTS:**

When an individual component section is focused in a one single frame; talkback says, “AMP Start UI Components Buttons”. This issues is observed for most of the components section from Links & Sharing. 

**EXPECTED RESULTS:**

Respective components sections should be categorized appropriately like other components. For. eg. Talkback says, “AMP Start UI Components Inputs” when Form Elements section is focused in a single frame.

**ATTACHMENT(S):**

[Components_Categories.mp4.zip](https://github.com/ampproject/ampstart/files/917487/Components_Categories.mp4.zip)
